src/color/Color.ts

```diff
@@ -0,0 +1,744 @@
+import { colord, extend } from 'colord';
```

Comment:
Would be good to have some tests with this new module since its used all over the place.
Might be something GPT can help with since it's pretty self contained?